optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessing.java

```diff
@@ -64,6 +73,9 @@ public Optional<OptOutInformation> createOptOutInformation(String information, l
             logger.log("Lambda can not parse the line: " + lineNumber);
             // The file line was parsed with an error
             optOutResultMap.put(lineNumber, new OptOutResult(new OptOutInformation(lineNumber, information), RecordStatus.REJECTED, ReasonCode.PARSE_ERROR));
+            if (!information.startsWith(FIRST_LINE) && !information.startsWith(LAST_LINE)) {
```

Comment:
var effectiveDate = convertToDate(information.substring(EFFECTIVE_DATE_INDEX_START, EFFECTIVE_DATE_INDEX_END));.. this will error out when imput is blank for any reason